site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -168,6 +170,14 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
       clearTimeout(showTimer);
     };
   }, [workspace, now, showAlertPendingInQueue]);
+
+  const showAutostartBanner =
+    (workspace.template_require_active_version ||
+      workspace.automatic_updates === "always") &&
+    workspace.autostart_schedule &&
+    workspace.outdated &&
+    !canAutostart;
```

Comment:
😵‍💫 maybe we could split this up into a couple constants? just a lot to decipher at once

```suggestion
  const updateRequired =
    (workspace.template_require_active_version ||
      workspace.automatic_updates === "always") && workspace.outdated;
  const autoStartFailing = workspace.autostart_schedule && !canAutostart;
  const requiresManualUpdate = updateRequired && autoStartFailing;
```